src/Entity/User.php

```diff
@@ -25,6 +26,9 @@ class User implements PasswordAuthenticatedUserInterface
     #[Column(type: Types::STRING, unique: true)]
     private string $email;
 
+    #[Column(type: Types::STRING)]
```

Comment:
It should accept only string role "admin, player, user". We can do the same thing with gender